src/main/resources/db/migration/V1__Create_Database.sql

```diff
@@ -0,0 +1,24 @@
+CREATE TABLE address (
+    id INTEGER PRIMARY KEY,
+    lines VARCHAR,
+    city VARCHAR,
+    country VARCHAR,
+    postcode VARCHAR,
+    international_postcode VARCHAR
+);
+
+CREATE TABLE user (
+    id INTEGER PRIMARY KEY,
+    first_name VARCHAR,
+    middle_name VARCHAR,
+    surname VARCHAR,
+    address INTEGER,
+    date_of_birth DATE,
+    FOREIGN KEY(address) REFERENCES address(id)
+);
+
+CREATE TABLE verifiedPid (
+    pid VARCHAR PRIMARY KEY,
+    user INTEGER,
```

Comment:
My understanding of verifiedPid table is it has id -> which is primary key and pid -> which is alpha-numberic value and foreign key to user table.
Did I misunderstood this ?

Reply:
From what I remember, the original plan was that but on discussing I think we said that since there is no way a pid can be linked to more than one user there was no reason to not make it the key. I might be wrong there though?

Reply:
Then what does user (INTEGER) represent here?